Formula/sratoolkit.rb

```diff
@@ -37,93 +28,33 @@ class Sratoolkit < Formula
   head do
     url "https://github.com/ncbi/sra-tools.git", branch: "master"
 
-    resource "ngs-sdk" do
-      url "https://github.com/ncbi/ngs.git", branch: "master"
-    end
-
     resource "ncbi-vdb" do
       url "https://github.com/ncbi/ncbi-vdb.git", branch: "master"
     end
   end
 
   depends_on "cmake" => :build
-  # Failed to build with `hdf5` at ncbi-vdb-source/libs/hdf5/hdf5dir.c:295:89:
-  # error: too few arguments to function call, expected 5, have 4
-  # herr_t h5e = H5Oget_info_by_name( self->hdf5_handle, buffer, &obj_info, H5P_DEFAULT );
-  # Try updating to `hdf5` on future release.
-  depends_on "hdf5@1.10"
+  depends_on "hdf5"
   depends_on "libmagic"
 
-  uses_from_macos "perl" => :build
   uses_from_macos "libxml2"
 
   def install
-    libxml2_prefix = if OS.mac?
-      MacOS.sdk_path/"usr"
-    else
-      Formula["libxml2"].opt_prefix
-    end
-    with_formula_args = %W[
-      --with-hdf5-prefix=#{Formula["hdf5@1.10"].opt_prefix}
-      --with-magic-prefix=#{Formula["libmagic"].opt_prefix}
-      --with-xml2-prefix=#{libxml2_prefix}
-    ]
-
-    ngs_sdk_prefix = buildpath/"ngs-sdk-prefix"
-    resource("ngs-sdk").stage do
-      cd "ngs-sdk" do
-        system "./configure", "--prefix=#{ngs_sdk_prefix}",
-                              "--build=#{buildpath}/ngs-sdk-build"
-        system "make"
-        system "make", "install"
-      end
-    end
-
-    ncbi_vdb_source = buildpath/"ncbi-vdb-source"
-    ncbi_vdb_build = buildpath/"ncbi-vdb-build"
-    ncbi_vdb_source.install resource("ncbi-vdb")
-    cd ncbi_vdb_source do
-      # Fix detection of hdf5 library on macOS as Apple Clang linker doesn't
-      # allow mixing static (-Wl,-Bstatic) and dynamic (-Wl,-Bdynamic) libraries
-      inreplace "setup/konfigure.perl", "-Wl,-Bstatic -lhdf5 -Wl,-Bdynamic", "-lhdf5" if OS.mac?
-
-      # Fix Linux error: `pshufb' is not supported on `generic64.aes'
-      # Upstream ref: https://github.com/ncbi/ncbi-vdb/issues/14
-      inreplace "libs/krypto/Makefile", "-Wa,-march=generic64+aes", "" if OS.linux?
-
-      system "./configure", "--prefix=#{buildpath}/ncbi-vdb-prefix",
```

Comment:
Upstream actually still recommends using `configure` on Linux/macOS; however, I think there are more problems using Perl-based (non autoconf) `configure` script given the non-standard behavior.

Directly running `cmake` is mainly recommended for Xcode and Visual Studio, but it seems to still work.

For now, hoping to reduce the number of workarounds by using `cmake` directly even if it isn't the recommended method.

Ref: https://github.com/ncbi/sra-tools/wiki/Building-and-Installing-from-Source#linux-macos-gmake